tests/e2e/test_io_performance.py

```diff
@@ -50,11 +38,19 @@ class TestIOPerformance(ManageTest):
         ]
     )
     def test_run_io(
-        self, size, io_direction, jobs, runtime, depth, sheet_index
+        self,
+        size,
+        io_direction,
+        jobs,
+        runtime,
+        depth,
+        sheet_index,
+        rbd_pod_factory
     ):
         """
         Test IO
         """
+        self.pod_obj = rbd_pod_factory()
```

Comment:
@fbalak @clacroix12 , this will create the pod as part of the test?

Reply:
Yes, by executing the method returned by the factory fixture within the test it will create the resource within the test itself and not in the setup or teardown.

Reply:
Then this is not good.
It changes the logic.
The test should start with a pod from setup